packages/vats/package.json

```diff
@@ -55,9 +55,8 @@
     "src/",
     "scripts/",
     "tools/",
-    "*.json",
-    "globals.d.ts",
-    "exported.js"
+    "index.*",
+    "exported.*"
```

Comment:
is that normally necessary? surely `index.js` is included in packages by default

Reply:
The files pointed to by `main` and `bin` are supposed to be automatically includer, but [apparently there are not](https://www.npmjs.com/package/@agoric/vats/v/0.15.2-u13.0?activeTab=code). Maybe another `yarn` gremlin?